packages/orbit-components/src/primitives/TooltipPrimitive/TooltipPrimitive.stories.tsx

```diff
@@ -12,7 +12,7 @@ import ListItem from "../../List/ListItem";
 import Heading from "../../Heading";
 import Button from "../../Button";
 import RenderInRtl from "../../utils/rtl/RenderInRtl";
-import { PLACEMENTS } from "../../common/consts";
```

Comment:
Now this could stay the same (maybe just changing the path if we feel strongly about it?)